package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "stylelint-config-cssrecipes",
+  "version": "1.0.0",
+  "description": "cssrecipes shareable config for stylelint",
+  "keywords": [
+    "browser",
+    "style",
+    "css",
+    "css-components",
+    "css-recipes",
+    "cssrecipes",
+    "recipes",
+    "linter"
```

Comment:
See this discussion why we choose to use a hyphen https://github.com/postcss/postcss/issues/319